controllers/machine_controller_phases_test.go

```diff
@@ -59,7 +60,7 @@ func TestReconcileMachinePhases(t *testing.T) {
 	defaultMachine := clusterv1.Machine{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      "machine-test",
-			Namespace: "default",
+			Namespace: metav1.NamespaceDefault,
```

Comment:
Should we use a random generated namespace instead?
(Same for similar changes in this file)

Reply:
In that case it would be just randomly generated string. Because we don't need actual Namespace object, as this test uses `fake.Client`.  I'm not sure of any benefits doing that, since each fake client instance has it's own storage for object, so there shouldn't be any conflicts.  Should we still generate random namespace name in that case?

Reply:
ok, thanks for providing context